cartoframes/data/enrichment/enrichment_service.py

```diff
@@ -22,9 +24,9 @@ def enrich(query_function, **kwargs):
     data_copy = _prepare_data(kwargs['data'], kwargs['data_geom_column'])
     tablename = _upload_dataframe(bq_client, user_dataset, data_copy, kwargs['data_geom_column'])
 
-    query = _enrichment_query(user_dataset, tablename, query_function, **kwargs)
+    queries = _enrichment_query(user_dataset, tablename, query_function, **kwargs)
```

Comment:
I would rename method to: `_enrichment_queries`

Reply:
You are right!